pages/common/serverless.md

```diff
@@ -0,0 +1,27 @@
+# serverless
```

Comment:
You've called the page `serverless`, but the command name is `sls`? We generally prefer pages to be filed under their command names.

Reply:
So this one is a little tricky I guess. The package is called [serverless](https://serverless.com), and it comes with two command aliases, `serverless` and `sls`. `sls` is shorter to type so I tend to go with that and I figured a good "tldr" of the command would show that usage as well. 

I see three options for resolving this:
1) Renaming all commands to `serverless` i.e. `serverless create`
2) Adding a description at the top indicating that it can be run as either `serverless` or `sls`.
3) Renaming the page `sls` (this one is my least favorite, I think it'll be confusing).
4) Having two pages, one called sls.md and one serverless.md with the same commands but using the different aliases. 

Is there one of these (or an option I couldn't think of) that you feel is more in line with convention?

Reply:
Please do 1 and 2. You can add a second line saying that the command can be run either as `serverless` or `sls`.

Reply:
Ok changed, let me know what you think. Thanks for the feedback y'all

Reply:
Yeah, #1 and #2 sound like the best plan to me.